source/MRViewer/MRRibbonMenuSearch.h

```diff
@@ -74,6 +78,7 @@ class MRVIEWER_CLASS RibbonMenuSearch
     bool mainInputFocused_ = false;
     bool blockSearchBtn_ = false;
     bool setInputFocus_ = false;
+    std::function<std::string( const std::shared_ptr<RibbonMenuItem>& )> requirementsFunc_;
```

Comment:
mb use alias for this function type everywhere it is used?

Reply:
fixed